php-checks/src/main/java/org/sonar/php/checks/security/RequestContentLengthCheck.java

```diff
@@ -145,8 +145,7 @@ private String getMethodName(FunctionCallTree tree) {
 
     if (callee.is(Tree.Kind.CLASS_MEMBER_ACCESS) && (((MemberAccessTree) callee).object().is(Tree.Kind.NAMESPACE_NAME))) {
       className = getFullyQualifiedName((NamespaceNameTree) ((MemberAccessTree) callee).object()).toString();
-    } else if (callee.is(Tree.Kind.OBJECT_MEMBER_ACCESS) && (((MemberAccessTree) callee).object() instanceof VariableIdentifierTreeImpl)) {
-      VariableIdentifierTreeImpl receiver = (VariableIdentifierTreeImpl) ((MemberAccessTree) callee).object();
+    } else if (callee.is(Tree.Kind.OBJECT_MEMBER_ACCESS) && (((MemberAccessTree) callee).object()instanceof VariableIdentifierTreeImpl receiver)) {
```

Comment:
```suggestion
    } else if (callee.is(Tree.Kind.OBJECT_MEMBER_ACCESS) && (((MemberAccessTree) callee).object() instanceof VariableIdentifierTreeImpl receiver)) {
```
I wonder why it's not fixed by spotless